package.json

```diff
@@ -69,6 +69,7 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "cross-env": "^7.0.3",
```

Comment:
My main concern is this hasn't been updated for two years even so it seems quite popular.

Reply:
I think it's ok, I'll approve.

Reply:
Yeah, great point. I think it's okay because the maintainer seems to be fairly responsible and says they will keep on top of security updates (https://github.com/kentcdodds/cross-env/issues/257), plus the number of dependencies it pulls in directly and indirectly is relatively low.